scripts/install_grpc

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# This script is only intended to run on the CI machines. Not for local
+# development.
+
+if [ "" == "$1" ] || [ "" == "$2" ] || [ "" == "$3" ] ; then
+   echo "error: invalid arguments"
+   echo "Usage: $0 <install-location> <version> <build-dir>"
+   echo ""
+   exit 1
+fi
+
+set -e
+set -x
+
+INSTALL_DIR=$1
+VERSION=$2
+BUILD_DIR=$3
+
+# Script may be executed in one of 3 modes on the CI:
+#
+# - No pre-existing build directory (DO_BUILD=1).
+#
+# - Pre-existing build directory, which must come from a cached successful
+# - build (DO_BUILD=0).  In this case, it can be direcetly installed
+
+if [ -d "${BUILD_DIR}" ] ; then
+    DO_BUILD=0
+else
+    DO_BUILD=1
+fi
+
+mkdir -p ${BUILD_DIR}
+pushd ${BUILD_DIR}
+
+    # Clone repo and submodules (if DO_BUILD == 1)
+    if [ "1" == "${DO_BUILD}" ] ; then
+        git clone --depth 1 -b ${VERSION} https://github.com/grpc/grpc .
+        git submodule update --depth 1 --init --recursive
+    fi
+
+    # Install protobuf
+    pushd third_party/protobuf
+        if [ "1" == "${DO_BUILD}" ] ; then
```

Comment:
Let's switch `[ "1" == "${DO_BUILD}" ]` to `[  "${DO_BUILD}"  == "1" ]` to be consistent in how we structure conditionals (see: https://github.com/clearmatics/zeth/pull/73#discussion_r335921994)
Also, in the files below we have things like  `[ $x == 10 ]` (variable on the left)

Reply:
switched conditional order